### PR TITLE
Add IsDecreasing benchmark

### PR DESCRIPTION
**Pull request by NgaiJustin**

### CODE REVIEW

File: benchmarks/core/is-decreasing.bril [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Input: an array of length 6
```
Comment (sampsyo) [inline benchmarks/core/is-decreasing.bril:1]: Looks like this may be out of date? The input appears to be an integer, not an array.

**Review by sampsyo — COMMENTED** [review]
Overall looking good! I found one possibly out of date comment, and it would be awesome if you could help resolve the merge conflict.

File: benchmarks/core/is-decreasing.bril (hunk 1 above)
Comment (NgaiJustin) [reply benchmarks/core/is-decreasing.bril]: Oops good catch. Just fixed this!

**Comment by sampsyo** [comment]
Excellent!

**Pull request merged**